### PR TITLE
Grant permissions and ignore exception from removePortForward

### PR DESCRIPTION
**Pull request by sravanmedarapu**
- Ignore if removePortForward throws error

- Call to grant permissions

https://github.com/appium/appium-adb/pull/189

@imurchie and @jlipps

### CODE REVIEW

**Comment by jlipps** [comment]
@sravanmedarapu is there an issue that has reported this error? just want to link it if so

**Comment by sravanmedarapu** [comment]
@jlipps No issue reported, felt adding a catch would be more appropriate. even failing `removePortForward` will not effect the end user.

**Comment by jlipps** [comment]
sounds good

File: lib/driver.js [hunk 1]
```diff
@@ -282,6 +282,13 @@ class AndroidUiautomator2Driver extends BaseDriver {
     if (this.opts.app) {
       await helpers.installApkRemotely(this.adb, this.opts);
     }
+    if (this.opts.autoGrantPermissions) {
+      try {
+        this.adb.grantAllPermissions(this.opts.appPackage, this.opts.app);
```
Comment (jlipps) [inline lib/driver.js]: shouldn't this be `await`ed?

**Comment by jlipps** [comment]
don't forget to publish :-)

**Comment by sravanmedarapu** [comment]
Sure :)

**Comment by sravanmedarapu** [comment]
published in 0.2.2

**Pull request merged**